lib/Doctrine/Annotations/Type/ScalarType.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Annotations\Type;
+
+/**
+ * @internal
+ */
+interface ScalarType extends Type
```

Comment:
This will be really confusing when coexisting with real ScalarType class

Reply:
No such class exists and I don't currently plan adding it (it's a union after all). If it does, we can rename stuff. It's internal after all.

Reply:
Fair enough